Add support for loading env values (#23)
AmericanAirlines/simple-env: README.md

> **NOTE**: if you choose to assign `optional` and `required` env vars individually, `setEnv` should only be done _once_ for each or you will overwrite your previously defined values.

### Loading .env files

If you set a path to an `.env` file, `simple-env` will parse the file and import the contents into the environment automatically. These will be available in the `process.env` object.

> > If you don't specify a path, `simple-env` **will not** automatically load any `.env` files. This is to avoid a breaking change.

Reviewer: ```suggestion
> :warning: **NOTE**: If you don't specify a path, `simple-env` **will not** automatically load any `.env` files.
```

```typescript
// src/env.ts
import setEnv from '@americanairlines/simple-env';

export const env = setEnv({
  required: {
    nodeEnv: 'NODE_ENV',
    someRequiredSecret: 'SOME_REQUIRED_SECRET',
  },
  options: {

>     envPath: "./.env"

Reviewer: This should work as well. Do you think the original is better?
```suggestion
    envPath: ".env"
```

  }
});
```

> Each variable needs to be declared on a separate line.

Reviewer: Let's add all of this to a new sub-section called `Correct .env Syntax` (or something like it). To keep the README simple, can we add all of that to a [`<details>` tag](https://www.w3schools.com/tags/tag_details.asp)?
